frontend/src/metabase/dashboard/containers/AutomaticDashboardApp.styled.tsx

```diff
@@ -22,3 +24,8 @@ export const SidebarRoot = styled.div`
 export const SidebarHeader = styled.h2`
   padding: 0.5rem 0;
 `;
+
+export const XrayIcon = styled(Icon)`
+  color: ${color("accent4")};
+  margin-right: 1rem;
```

Comment:
should we try to use `${space(2)}` or hardcode value is okay? I always try to use `space` as I thought it should be the way to go. What do you think?

Reply:
I have mixed thoughts about `space`. The problem is that it has nothing to do with our design system. `space(0...4)` constants were simply invented by an engineer. Designers don't follow these conventions.

Reply:
`colors` is a good example of a thing that is used by our designers. These values actually have corresponding values in Figma mockups and that's why it works and `space` kind of doesn't.

Reply:
but using `space` also means it's easier in the future if we will implement the space from our design system (not even sure if we have one, but I guess it's common enough in most design systems).